packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -65,3 +70,22 @@
 .edit-site-screen-background-color__control {
 	padding: $grid-unit-20;
 }
+
+.edit-site-global-styles-variations_item {
+	box-sizing: border-box;
+	padding: 3px;
+	border-radius: $radius-block-ui;
+	border: $gray-200 1px solid;
+
+	&.is-active {
+		border: $gray-900 1px solid;
+	}
+
+	&:hover {
+		border: var(--wp-admin-theme-color) 1px solid;
+	}
+
+	&:focus {
+		border: var(--wp-admin-theme-color) 1px solid;
```

Comment:
Should we use $border-width-focus which is 2px?

Reply:
The problem if we change the border width on focus is that it creates jumps with the other states.